TodoList.Web/Services/EmailSender.cs

```diff
@@ -42,10 +44,9 @@ public async Task SendEmailAsync(string email, string subject, string message)
                     throw new Exception("The email couldn't be sent.");
```

Comment:
I'd suggest refactoring this a bit. Instead of throwing an exception and catching it a few lines below: just log instantly that the e-mail couldn't be sent. Throwing exceptions is expensive for performance, so try to prevent it. Of course the catch block can stay, for other exceptions.

Reply:
The throw is removed and instead logging added.